docs/db_ops.md

```diff
@@ -97,3 +97,13 @@ typed; this is necessary for sensitive data, such as passwords.
 This method will is used to support verbose dialog. If `obj` is supplied, it is
 printed after `string`. If `force` is `True`, then the message will print
 regardless of the value in `self.verbose`.
+
+#### `bool self._write_table(df : DataFrame, conflict_columns=None : list of string)`
+    Write the given dataframe into the database. The DataFrame is expected to
+    be well formed by the subclass, and as such should only be called by a
+    subclass.
+    conflict_columns is a list of string which specifies which columns should
+    trigger the ON CONFLICT condition. ON CONFLICT is triggered when values
+    in the specified columns already exist on the table, and will do nothing
+    (to avoid an error, as postgres will throw a fit when a duplicate row is
+    written onto the table).
```

Comment:
The rest of the file doesn't use the code indentation for the method documentation, so if you could change that, I'd appreciate it.

Reply:
Oops haha